zipkin-cassandra/src/main/scala/com/twitter/zipkin/storage/cassandra/CassandraSpanStore.scala

```diff
@@ -35,6 +35,7 @@ object CassandraSpanStoreDefaults {
   val SpanTtl = 7.days
   val IndexTtl = 3.days
   val MaxTraceCols = 100000
+  val MaxConnections = 2
```

Comment:
Other C* people can chime in too, but I'd like to understand better why the OP ended up using what the driver thought were REMOTE nodes. In C* these typically are the nodes that are in a whole different datacenter (potentially across the US, for example).